en_ewt-ud-test.conllu

```diff
@@ -1109,26 +1109,26 @@
 16	respect	respect	NOUN	NN	Number=Sing	11	nmod	11:nmod:with	_
 17	to	to	ADP	IN	_	18	case	18:case	_
 18	Parts	part	NOUN	NNS	Number=Plur	16	nmod	16:nmod:to	_
-19	I	i	NUM	CD	NumType=Card	18	nummod	18:nummod	_
+19	I	I	NUM	CD	NumForm=Roman|NumType=Card	18	nummod	18:nummod	_
 20	through	through	ADP	IN	_	21	case	21:case	_
-21	IV	iv	NUM	CD	NumType=Card	19	nmod	19:nmod:through	SpaceAfter=No
+21	IV	IV	NUM	CD	NumForm=Roman|NumType=Card	19	nmod	19:nmod:through	SpaceAfter=No
 22	,	,	PUNCT	,	_	23	punct	23:punct	_
-23	VI	vi	NUM	CD	NumType=Card	19	conj	18:nummod|19:conj:and	_
+23	VI	VI	NUM	CD	NumForm=Roman|NumType=Card	19	conj	18:nummod|19:conj:and	_
 24	through	through	ADP	IN	_	29	case	29:case	_
-25	VI	vi	NUM	CD	NumType=Card	27	compound	27:compound	SpaceAfter=No
+25	VI	VI	NUM	CD	NumForm=Roman|NumType=Card	27	compound	27:compound	SpaceAfter=No
 26	-	-	PUNCT	HYPH	_	27	punct	27:punct	SpaceAfter=No
 27	D	D	NOUN	NN	Number=Sing	29	compound	29:compound	SpaceAfter=No
 28	-	-	PUNCT	HYPH	_	29	punct	29:punct	SpaceAfter=No
-29	iii	iii	NUM	CD	NumType=Card	23	nmod	23:nmod:through	SpaceAfter=No
+29	iii	III	NUM	CD	NumForm=Roman|NumType=Card	23	nmod	23:nmod:through	SpaceAfter=No
```

Comment:
Actually I don't think the lemma should be capitalized. In the text there is a clear semantic distinction between uppercase and lowercase roman numerals. So if we're preserving the roman/decimal distinction in the lemmas, might as well preserve capitalization too.